### ISSUE TITLE
How to install dotnet core 1.1.0?

### ISSUE THREAD

**Issue by MuthiyalLakshmi** [dotnet/cli#5594]
![screenshot 8](https://cloud.githubusercontent.com/assets/25578966/22638591/e2f72416-ebfd-11e6-9870-e2ff0bab2a22.png)

**Comment by MuthiyalLakshmi** [comment]
Please Anyone clear my doubt.
I tried a lot but its not working.

**Comment by piotrpMSFT** [comment]
https://www.microsoft.com/net/download/core#/current/runtime

**Comment by piotrpMSFT** [comment]
@MuthiyalLakshmi I hope the link helps. 

@livarcocc @blackdwarf we need to better manage this experience in 2.0. Nothing's really preventing us from providing a link... or offering to download & install the framework.

**Comment by livarcocc** [comment]
Yes. We have talked about it a couple of times. How do you want to track that improvement?

**Comment by MuthiyalLakshmi** [comment]
sry its not working having the same problem showing again and again

**Comment by blackdwarf** [comment]
@MuthiyalLakshmi on the link that @piotrpMSFT shared you should be able to find an installer for the 1.1.0 runtime. Install that and try to rerun. It should clear up the error message.

@livarcocc the issue should be in the core-setup repo, right? This message is from the host? We should still leave this one around for tracking.

**Comment by livarcocc** [comment]
@blackdwarf Yes, the message is in the host, however, maybe the CLI could augment this experience somehow, like, offering to install the missing shared framework.

**Comment by blackdwarf** [comment]
@livarcocc this is a part of the overall plans for 2.0; if we have a CLI gesture for the user to install a runtime from the CLI command, then yes, that piece should be lit up here as well. We need a specc. Imagine what I will be doing. :)

**Comment by livarcocc** [comment]
We have been talking about a `dotnet install runtime` command that would address a big chunk of this.
